frontend/src/i18n/locales/en-US.json

```diff
@@ -68,6 +68,7 @@
     "question-circle-warning": "Don't use this feature unless you know what you're doing.",
```

Comment:
Still getting one error when I attempt this on Optimism :cry: 

![image](https://user-images.githubusercontent.com/1479802/218203732-3e99ace1-eea6-4a4a-a8be-fb2cf0e9a111.png)